tracing/tracing.go

```diff
@@ -94,6 +108,11 @@ func (r *responseWriter) WriteHeader(statusCode int) {
 	r.ResponseWriter.WriteHeader(statusCode)
 }
 
+func (r *responseWriter) Write(b []byte) (int, error) {
+	r.contentLength += int64(len(b))
```

Comment:
Since now we are messing with writes we could add a response body on the test (from what I can remember it is a simple status code that is tested).